Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

sql/mysqld.cc:6915:30: error: use of undeclared identifier 'TRUE'
    my_getopt_skip_unknown = TRUE;
                             ^

--- sql/mysqld.cc.orig	2020-06-16 16:31:03 UTC
+++ sql/mysqld.cc
@@ -6910,7 +6912,7 @@ int mysqld_main(int argc, char **argv)
   if (opt_keyring_migration_source || opt_keyring_migration_destination ||
       migrate_connect_options) {
     Migrate_keyring mk;
-    my_getopt_skip_unknown = TRUE;
+    my_getopt_skip_unknown = true;
     if (mk.init(remaining_argc, remaining_argv, opt_keyring_migration_source,
                 opt_keyring_migration_destination, opt_keyring_migration_user,
                 opt_keyring_migration_host, opt_keyring_migration_password,
